src/components/Feed.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+
+function Feed({ requests, receiveSelectRequest }) {
+  function passClick(selection) {
+    receiveSelectRequest(selection);
+    console.log("hello");
+  }
+  return (
+    <main className="feed">
+      <ul>
+        {requests ? (
```

Comment:
when there are no requests it would be better to store it as an empty array and check the length property of the array to check if there is no data